docker/ngc_overview_hugectr.md

```diff
@@ -0,0 +1,101 @@
+
+## Getting Started
+
+### Launch merlin-hugectr Container
+
+You can pull the training containers with the following command:
+
+```
+docker run --runtime=nvidia --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_NICE nvcr.io/nvidia/merlin/merlin-hugectr:latest /bin/bash
+```
+
+If you are running on a docker version 19+, change --runtime=nvidia to --gpus all.
+The container will open a shell when the run command completes execution, you will be responsible for starting the jupyter lab on the docker container. Should look similar to below:
+
+```
+root@2efa5b50b909:
+```
+
+Install jupyter-lab with `conda` or `pip`: [Installation Guide](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html)
+
+```
+pip install jupyterlab
+```
+
+Finally start the jupyter-lab server:
+
+```
+jupyter-lab --allow-root --ip='0.0.0.0' --NotebookApp.token=''
+```
+
+Now you can use any browser to access the jupyter-lab server, via :8888
+Once in the server, navigate to the /nvtabular/ directory and explore the code base or try out some of the examples.
+Within the container is the codebase, along with all of our dependencies, particularly RAPIDS Dask-cuDF. The easiest way to get started is to simply launch the container above and explore the examples within.
+
+### Other NVIDIA Merlin containers
+
+Merlin containers are available in the NVIDIA container repository at the following locations:
+Table 1: Merlin Containers
+
+| Container name | Container location | Functionality |
+|----------------|--------------------|---------------|
+| merlin-hugectr | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-hugectr | Merlin and HugeCTR |
+| merlin-pytorch | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-pytorch | Merlin and PyTorch |
+| merlin-hugectr | https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-tensorflow | Merlin and TensorFlow |
+
+## Examples and Tutorials
+
+We provide a collection of examples, use cases, and tutorials for [NVTabular](https://github.com/NVIDIA/NVTabular/tree/main/examples) and [HugeCTR](https://github.com/NVIDIA/HugeCTR/tree/master/notebooks) as Jupyter notebooks in our repository. These Jupyter notebooks are based on the following datasets:
+- MovieLens
+- Outbrain Click Prediction
+- Criteo Click Ads Prediction
+- RecSys2020 Competition Hosted by Twitter
+- Rossmann Sales Prediction
+  With the example notebooks we cover the following:
+- Preprocessing and feature engineering with NVTabular
+- Advanced workflows with NVTabular
+- Accelerated dataloaders for TensorFlow and PyTorch
+- Scaling to multi-GPU and multi nodes systems
+- Integrating NVTabular with HugeCTR
+- Deploying to inference with Triton
+
+For more sample models and their end-to-end instructions for HugeCTR visit the link: https://github.com/NVIDIA/HugeCTR/tree/master/samples
+
+## Learn More
```

Comment:
Good heading.

My sugg is to begin the section with documentation URLs:
<https://nvidia-merlin.github.io/NVTabular/main/Introduction.html>

<https://nvidia-merlin.github.io/HugeCTR/master/hugectr_user_guide.html>

...and then just make sure that we offer links to the repositories on GitHub.